services/outputhost/cgcache.go

```diff
@@ -308,6 +317,11 @@ func (cgCache *consumerGroupCache) loadExtentCache(destType shared.DestinationTy
 		cgCache.cgMetrics.Increment(load.CGMetricNumOpenExtents)
 		cgCache.hostMetrics.Increment(load.HostMetricNumOpenExtents)
 
+		// get the initial credits based on the message cache size
+		cfg, err := cgCache.getDynamicCgConfig()
+		if err == nil {
+			extCache.initialCredits = cgCache.getMessageCacheSize(cfg, defaultNumOutstandingMsgs)
```

Comment:
We need to set extCache.initialCredits to a default value if the err != nil, right?